tests/lib/snaps/test-snapd-netlink-connector/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-snapd-netlink-connector
+summary: Basic snap to bind the netlink connector
+description: A basic snap used to bind the netlink connector
+version: 1.0
+
+apps:
+  bind:
+    command: bin/bind
+    plugs: [netlink-connector, network, network-control]
```

Comment:
I'm curious why do you need the other interfaces, netlink-connector should be self-sufficient, CC @jdstrand

Reply:
@zyga Me too, I needed the to plug network to create the socket and the network-control to bind the socket.

Reply:
@zyga - it should be self-sufficient, but is not.

@sergiocazzolato - as part of this PR, can you add this to the interface:
```
const netlinkConnectorConnectedPlugAppArmor = `
# Description: Can use netlink to communicate with kernel connector. Because
# NETLINK_CONNECTOR is not finely mediated and app-specific, use of this
# interface allows communications via all netlink connectors.
# https://github.com/torvalds/linux/blob/master/Documentation/connector/connector.txt
network netlink,
# CAP_NET_ADMIN required per 'man 7 netlink'
capability net_admin,
`
...
		connectedPlugAppArmor: netlinkConnectorConnectedPlugAppArmor,
...
```

With the above, you shouldn't need to plugs anything but netlink-connector.

Reply:
The same will be needed for netlink-audit.